pkgs/tools/networking/spoof-mac/default.nix

```diff
@@ -13,8 +13,7 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ docopt ];
 
-  # No tests
-  doCheck = false;
```

Comment:
You can keep this line, it doesn't influence `pythonImportsCheck` and AFAIK it is faster since it doesn't run a `checkPhase`: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck